libs.ts

```diff
@@ -699,7 +699,7 @@ export const libs: Lib[] = [
 		name: 'disnake',
 		url: 'https://github.com/DisnakeDev/disnake',
 		language: 'Python',
-		apiVer: 8,
+		apiVer: 9,
```

Comment:
looks like this change isn't released yet?
```suggestion
		apiVer: '8 stable, 9 dev',
```

Reply:
Pycord isn't using v9 in the stable version (they don't have anything in stable but anyway) and yet it's been mentioned as "stable". So, would the same logic apply to api version here?

Reply:
> Pycord isn't using v9 in the stable version (they don't have anything in stable but anyway) and yet it's been mentioned as "stable". So, would the same logic apply to api version here?

Probably worth making an issue, I didn't see this until you pointed it out.

Reply:
if there aren't visible stable and dev versions then the version that exists is considered stable